src/RestControllers/FhirPatientRestController.php

```diff
@@ -78,7 +78,9 @@ public function getAll($search)
 
         $searchParam = array(
             'name' => $search['name'],
-            'dob' => $search['birthdate']
+            'dob' => $search['birthdate'],
```

Comment:
to support $search['birthdate'], looks like also need to incorporate that in conditional at line 227 in /src/Services/PatientService.php

Reply:
In FhirPatientRestController.php, the $search['birthdate'] is assigned to 'dob' in the $searchParam array. 'dob' has been included in the 'if' condition on line 227 of PatientService.php.

Reply:
oops, thanks for pointing that out